### PR TITLE
Move to new OAuth Backend

### PR DESCRIPTION
**Pull request by tkw1536**
This commit updates the People Client to work with the new dreamjub
backend, using OAuth and updated endpoints.

### CODE REVIEW

File: js/stalk.js [hunk 1]
```diff
@@ -16,6 +16,12 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 var c;
+var dreamjub_url = 'http://localhost:8000/';
+var dreamjub_client_id = 'uXDaB3j1n8DiOU8WXjwCa2gbKqLveieoIDpjFa1d';
+
+// var dreamjub_url = 'https://leadership.jacobs.university/';
```
Comment (kuboschek) [inline js/stalk.js:22]: Should this be pointing to the staging version? Not sure.

Comment (tkw1536) [reply js/stalk.js:22]: Well, it was a comment for testing.

Comment (tkw1536) [reply js/stalk.js:22]: In production, it actually points to the jacobs.university instance, but I didn't want to put the key in this repository.

Comment (kuboschek) [reply js/stalk.js:22]: Fair enough.

**Review by kuboschek — APPROVED** [review]
let's do this.

**Pull request merged**